src/assets/scripts/client/InputController.js

```diff
@@ -765,6 +767,23 @@ export default class InputController {
         ]);
     }
 
+    /**
+     *
```

Comment:
Missing description

Reply:
I added a description but if you can come up with something better, go for it.